QGL/ControlFlow.py

```diff
@@ -5,25 +5,29 @@
 
 ## QGL control-flow statements ##
 
+# FIXME qif, qwhile, and qdowhile need an address to pass to CMP
```

Comment:
Note that another option is to simply deprecate these methods at the QGL1 layer since QGL2 exposes a much better interface.